tests/component/_task_modules/test_triggers.py

```diff
@@ -1,24 +1,38 @@
-from datetime import timedelta
+from datetime import timezone
 
 import pytest
-from hightime import datetime as ht_datetime
+from hightime import datetime as ht_datetime, timedelta as ht_timedelta
 
-from nidaqmx.constants import Timescale
+import nidaqmx
+from nidaqmx.constants import Edge, Timescale, TimestampEvent, TriggerType
+from nidaqmx.error_codes import DAQmxErrors
 from nidaqmx.task import Task
 
 
 @pytest.fixture()
 def ai_voltage_field_daq_task(task, sim_field_daq_device):
-    """Gets AI voltage task."""
+    """Gets AI voltage field daq task."""
     task.ai_channels.add_ai_voltage_chan(sim_field_daq_device.ai_physical_chans[0].name)
     yield task
 
 
+@pytest.fixture()
+def ci_count_edges_task(task) -> nidaqmx.Task:
+    chan = task.ci_channels.add_ci_count_edges_chan(
+        "cdaqChassisTester/_ctr0",
+    )
+    chan.ci_count_edges_term = "/cdaqChassisTester/te0/SampleClock"
+    chan.ci_count_edges_active_edge = Edge.RISING
+    return task
+
+
 def test___default_arguments___cfg_time_start_trig___no_errors(
     ai_voltage_field_daq_task: Task,
 ):
     ai_voltage_field_daq_task.timing.cfg_samp_clk_timing(1000)
-    trigger_time = ht_datetime.now() + timedelta(seconds=10)
+    utc_dt = ht_datetime.now(timezone.utc)  # UTC time
+    dt_now = utc_dt.astimezone()  # local time
+    trigger_time = dt_now + ht_timedelta(seconds=10)
```

Comment:
`trigger_time` is in local time. However, the interpreter class's [get_trig_attribute_timestamp](https://github.com/ni/nidaqmx-python/blob/master/generated/nidaqmx/_library_interpreter.py#L3806) doesn't specify a timezone. So how is `when_value` coming back in local time?

Reply:
To briefly summarize the whole thing, 
trigger time gets calculated as AbsoluteTime in the interpreter class's [from_datetime](https://github.com/ni/nidaqmx-python/blob/0905e65642ee934d011fc594d52829fce21d3fcd/generated/nidaqmx/_library_interpreter.py#L403C32-L403C45) which does a simple subtraction following the formula in my libtime PR
`seconds_since_1904 = int((dt - AbsoluteTime._EPOCH_1904) / ht_timedelta(seconds=1))`.
This, in itself, returns the correct number of seconds _**taking into consideration timezone difference**_. I checked this on an online compiler
![image](https://github.com/ni/nidaqmx-python/assets/74756143/5a29b60c-39a0-449b-9d05-bff6a56d517b)

So, timezone is accounted for when setting `trigger_time`. 

Then when reading it back, as you said it goes to [get_trig_attribute_timestamp](https://github.com/ni/nidaqmx-python/blob/master/generated/nidaqmx/_library_interpreter.py#L3806), which returns the value [**_in datetime_**](https://github.com/ni/nidaqmx-python/blob/0905e65642ee934d011fc594d52829fce21d3fcd/generated/nidaqmx/_library_interpreter.py#L3819C20-L3819C35). This to_datetime function will call `_convert_to_desired_timezone` when returning the values, as seen in my [lib_time PR](https://github.com/ni/nidaqmx-python/pull/504) here. 
![image](https://github.com/ni/nidaqmx-python/assets/74756143/fe216ed3-af8b-4f3f-8fb8-3d6de4265411)

This shows that we do use this timezone conversion at least in python.